base/strings/util.jl

```diff
@@ -77,8 +77,25 @@ julia> a = "March"
 
 julia> chop(a)
 "Marc"
+
+julia> chop(a, 1, 2)
+"ar"
+
+julia> chop(a, 5, 5)
```

Comment:
The description should mention that no error is thrown if there are not enough characters.

Reply:
Added. This is the current behavior as `chop("")` returns `""`.